data_tools/add.py

```diff
@@ -0,0 +1,65 @@
+import firebase_client_wrapper as fcw
+
+import json
+import sys
+
+import validate_code_scheme
+import validate_message_structure
+import validate_user_list
+
+# if (len(sys.argv) != 5):
+#     print ("Usage python add.py crypto_token dataset_id users|schemes|messages path")
+#     print ("add only adds data items that don't currently exist, by id. Existing data")
+#     print ("items remain untouched")
+#     exit(1)
+
+CRYPTO_TOKEN_PATH = sys.argv[1]
+fcw.init_client(CRYPTO_TOKEN_PATH)
+
+DATASET_ID = sys.argv[2]
+CONTENT_TYPE = sys.argv[3]
+PATH = sys.argv[4]
+
+dataset_ids = fcw.get_dataset_ids()
+
+if DATASET_ID not in dataset_ids:
+    print ("WARNING: dataset {} does not exist, this will create a new dataset".format(DATASET_ID))
```

Comment:
Do you think we should add here an await on key input from the console?
Ie. print a message like: Press y/n to continue/abort then wait for user input. Would that be too complicated for now?